python/reg_generator/pkg/setup.py

```diff
@@ -18,8 +18,34 @@ def getreqs():
         reqs = f.readlines()
         return [x.strip() for x in reqs]
 
+def getVersion():
```

Comment:
Note for future, should probably figure out a way to put all this common stuff into a function provider `import`ed from the `gembuild` submodule